rep-0151.rst

```diff
@@ -0,0 +1,151 @@
+REP: 151
+Title: Python 3 Support
+Author: Dirk Thomas
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 23-Dec-2017
+
+Outline
+=======
+
+#. Abstract_
+#. Motivation_
+#. Rationale_
+#. Proposal_
+#. Requirements_
+#. References_
+#. Copyright_
+
+Abstract
+========
+
+This REP describes the path for switching from Python 2 to Python 3 in ROS 1.
+
+Motivation
+==========
+
+As described in the Python release schedule [1]_ Python 2.7 will only receive
+bugfix releases until 2020.
+As a consequence Ubuntu is trying to demote Python 2.7 into the `universe`
+repository for 18.04 since they don't want to guarantee support for 5 years.
+While it is unlikely that Python 2.7 will be removed from the archives entirely
+it should not be relied on past its EOL data.
+Therefore ROS 1 needs to begin moving to Python 3 in order to make the
+transition as smooth as possible.
+
+Rationale
+=========
+
+The goal for a future ROS 1 distribution to support Python 3 should be achieved
+no later than for ROS O-turtle (Mai 2020).
```

Comment:
"Mai" -> "May"

Reply:
Thanks. Fixes spelling in 07cb35b6bd0b2f3e0757264b7d0535b97d78d7de.